ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/Genesis/Tests/LongRangeAttack.hs

```diff
@@ -23,17 +23,21 @@ import           Test.Util.Orphans.IOLike ()
 import           Test.Util.TestBlock (TestBlock, unTestHash)
 
 tests :: TestTree
-tests = testProperty "long range attack" prop_longRangeAttack
+tests =
+  testGroup "long range attack" [
+    testProperty "one adversary" (prop_longRangeAttack 1 [10]),
+    testProperty "three adversaries" (prop_longRangeAttack 1 [2, 5, 10])
+  ]
```

Comment:
What about one adversary with frequency 2? Or 1.5? How do they behave? Are they sufficient to make the test pass?

Reply:
I'd assume that too, but they probably need the prefix to be long enough or something and that might be a function of `k`?